medium/_06_fulltests/answers/merges.answer

```diff
@@ -28,7 +28,7 @@ count(*)
 
 ===================================================
 count(*)    
-1280     
+0     
```

Comment:
need to modify test case.

Reply:
comparing cannot be work in where statement
char varying collate iso88591_bin_ts = char default collation .